### ISSUE TITLE
npm module: Cannot find module './lib/formatters/prose'

### ISSUE THREAD

**Issue by misterflippy** [prantlf/gulp-jsonlint#1]
When I use the latest npm module (2.2.0), I get the following error output:
```
internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module './lib/formatters/prose'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (D:\Projects\minecraft\minecraft-bedrock-jsonschema\test\node_modules\@prantlf\gulp-jsonlint\index.js:120:10)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
```

I installed gulp-jsonlint with the following command:
```npm i -D @prantlf/gulp-jsonlint```

My gulpfile.js is:
```
const { src, dest } = require('gulp');
const jsonlint = require('@prantlf/gulp-jsonlint')

function validateTask() {
    return src('./test-files/behaviors/feature_rules/*.json')
        .pipe(jsonlint())
        .pipe(jsonlint.reporter())
}

exports.default = validateTask;
```

There is no `./lib/formatters/prose` file in `node_modules/@prantlf/gulp-jsonlint`. Adding the lib directory from the github repository resolved the error locally. (The code needs all the files in there)

**Comment by idleberg** [comment]
Same here. The [tarball](https://registry.npmjs.org/@prantlf/gulp-jsonlint/-/gulp-jsonlint-2.2.1.tgz) doesn't include the lib-folder, even though that's not in `.npmignore`. A fix would be appreciated!

**Comment by prantlf** [comment]
Thanks for the report, @misterflippy! I didn't notice it, because I used a local copy of `gulp-jsonlint` instead of installing it by `npm`...

It was caused by the combination of `.npmignore` which was ignored because of files` in `package.json`. I dropped the former in favour of the latter and released the fix in the version [2.3.0](https://github.com/prantlf/gulp-jsonlint/releases/tag/v2.3.0).
